portfolio/src/main/webapp/index.html

```diff
@@ -76,6 +76,8 @@ <h2 id="gettoknowme">Get To Know Me!</h2>
             <li><a href="#https://www.linkedin.com/in/sofia-fernandez-chacin-242787175/">LinkedIn</a>  <a 
             href="#https://github.com/sofifch">GitHub</a>  <a 
```

Comment:
The `#` looks like a mistake to me. Is that valid syntax?

Reply:
You are right, it is a mistake! Will fix it.